.github/workflows/veramo-website.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger veramo-website build
+on:
+  push:
```

Comment:
It would be nice to only run this if the publish workflow succeeds

Reply:
Remnants of experiments. Fixed.